modules/dpu-add-hosts-cluster.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/dpu-hardware-offload.adoc
+
+:_content-type: PROCEDURE
+[id="adding-additional-hosts-to-cluster_{context}"]
+= Adding additional hosts to an existing cluster
+
+Add worker nodes to the existing infrastructure cluster using the procedure outlined. This procedure 
+
+.Prerequisites
+
+* Shared VLAN: The assisted installer requires all nodes to be present on the same VLAN. This is a requirement of utilizing a virtual IP for both ingress and API.
+* DHCP: Much like an IPI installation DHCP is required in the VLAN above.
+* DNS: Records for ingress and API are required for accessing the cluster. The assisted installer supports creating these records for the user via route53, but in this article the following DNS records have been created.
+
+.Procedure
+
+. Import A Cluster and Register an InfraEnv
+
+.. Import 
++
+* POST /v2/clusters/import
+* operationId: v2ImportCluster
++
+[source,terminal]
+----
+curl -X POST -H "Content-Type: application/json" \
+    -d '{"name":"mycluster","openshift_cluster_id":"<id>","api_vip_dnsname":"api-vip.example.com"}' \
+    <HOST>:<PORT>/api/assisted-install/v2/clusters/import
+----
```

Comment:
@tsorya  we can change this process to the new Day2 Transform endpoint 

`<HOST>:<PORT>/api/assisted-install/v2/clusters/actions/add-worker-nodes/allow-add-workers`